diff --git a/ACE/ace/Asynch_Acceptor.h b/ACE/ace/Asynch_Acceptor.h
index 36b3aa1ca76..eaa2aae6f9b 100644
--- a/ACE/ace/Asynch_Acceptor.h
+++ b/ACE/ace/Asynch_Acceptor.h
@@ -234,6 +234,10 @@ protected:
    */
   virtual HANDLER *make_handler (void);
 
+  /// Address family used to open this object. Obtained from @a address passed
+  /// to @c open().
+  int addr_family_;
+
 private:
   /// Handle used to listen for new connections.
   ACE_HANDLE listen_handle_;
@@ -253,10 +257,6 @@ private:
 
   /// Bytes to be read with the <accept> call.
   size_t bytes_to_read_;
-
-  /// Address family used to open this object. Obtained from @a address passed
-  /// to @c open().
-  int addr_family_;
 };
 
 ACE_END_VERSIONED_NAMESPACE_DECL
