--- D:/projects/ATCD/ACE/ace/Module.h	Fri Aug 18 22:45:15 2017
+++ C:/Users/Erik/Desktop/Module.h	Fri Aug 18 22:33:13 2017
@@ -59,6 +59,8 @@
     /// Indicates that close() should not delete any Tasks.
     M_DELETE_NONE = 4
   };
+
+  inline virtual ~ACE_Module_Base (void) {};
 };
 
 /**
@@ -162,7 +164,7 @@
   ACE_Module<ACE_SYNCH_USE, TIME_POLICY> *next (void);
 
   /// Set the next pointer to the module above in the stream.
-  void next (ACE_Module<ACE_SYNCH_USE, TIME_POLICY> *m);
+  virtual void next (ACE_Module<ACE_SYNCH_USE, TIME_POLICY> *m);
 
   /// Dump the state of an object.
   void dump (void) const;
