--- C:\Users\Erik\OneDrive\Desktop\Task_T.h
+++ H:\lib\ACE_TAO\ACE\ace\Task_T.h
@@ -124,7 +124,7 @@
 
   // = Pointers to next ACE_Task_Base (if ACE is part of an ACE_Stream).
   /// Get next Task pointer.
-  ACE_Task<ACE_SYNCH_USE, TIME_POLICY> *next (void);
+  virtual ACE_Task<ACE_SYNCH_USE, TIME_POLICY> *next (void);
 
   /// Set next Task pointer.
   void next (ACE_Task<ACE_SYNCH_USE, TIME_POLICY> *);

