diff --git a/src/modules/CMakeLists.txt b/src/modules/CMakeLists.txt
index 64c6b1c6..7616f470 100644
--- a/src/modules/CMakeLists.txt
+++ b/src/modules/CMakeLists.txt
@@ -9,7 +9,7 @@ add_subdirectory (misc)
 add_subdirectory (ml)
 # *IMPORTANT NOTE*: comment temporarily to disambiguate ACEStream <-> ACENetwork
 #                   interdependencies
-add_subdirectory (net)
+#add_subdirectory (net)
 # *IMPORTANT NOTE* /END
 add_subdirectory (stat)
 add_subdirectory (vis)
diff --git a/test_i/CMakeLists.txt b/test_i/CMakeLists.txt
index 385c84dd..7185c7e0 100644
--- a/test_i/CMakeLists.txt
+++ b/test_i/CMakeLists.txt
@@ -293,12 +293,12 @@ add_subdirectory (commandspeech)
 add_subdirectory (extract_stream)
 # *IMPORTANT NOTE*: comment temporarily to disambiguate ACEStream <-> ACENetwork
 #                   interdependencies
-add_subdirectory (camstream)
-add_subdirectory (filestream)
-add_subdirectory (http_get)
-add_subdirectory (http_get_2)
-add_subdirectory (pop_receive)
-add_subdirectory (smtp_send)
+#add_subdirectory (camstream)
+#add_subdirectory (filestream)
+#add_subdirectory (http_get)
+#add_subdirectory (http_get_2)
+#add_subdirectory (pop_receive)
+#add_subdirectory (smtp_send)
 # *IMPORTANT NOTE* /END
 add_subdirectory (imagesave)
 add_subdirectory (mp3player)
@@ -309,10 +309,10 @@ add_custom_target (test_i ALL
 add_dependencies (test_i avsave camera_ar camera_ml camera_msa commandspeech extract_stream imagesave mp3player speechcommand)
 # *IMPORTANT NOTE*: comment temporarily to disambiguate ACEStream <-> ACENetwork
 #                   interdependencies
-add_dependencies (test_i camsource camtarget filesource filetarget HTTP_get_1 HTTP_get_2 POP_receive SMTP_send)
-if (WIN32)
- add_dependencies (test_i camtarget_source camtarget_source_mf)
-endif (WIN32)
+#add_dependencies (test_i camsource camtarget filesource filetarget HTTP_get_1 HTTP_get_2 POP_receive SMTP_send)
+#if (WIN32)
+# add_dependencies (test_i camtarget_source camtarget_source_mf)
+#endif (WIN32)
 # *IMPORTANT NOTE* /END
 
 cmake_policy (SET CMP0012 NEW)
diff --git a/test_u/CMakeLists.txt b/test_u/CMakeLists.txt
index ac92b770..d50b88c5 100644
--- a/test_u/CMakeLists.txt
+++ b/test_u/CMakeLists.txt
@@ -312,7 +312,7 @@ add_subdirectory (document)
 add_subdirectory (filecopy)
 # *IMPORTANT NOTE*: comment temporarily to disambiguate ACEStream <-> ACENetwork
 #                   interdependencies
-add_subdirectory (http_get)
+#add_subdirectory (http_get)
 # *IMPORTANT NOTE* /END
 add_subdirectory (imagescreen)
 add_subdirectory (parser)
@@ -325,7 +325,7 @@ add_custom_target (test_u ALL
 add_dependencies (test_u animated_gif audioeffect branch camera_filter camerascreen camsave capturewindow document filecopy imagescreen parser QR_decode RIFF_decoder)
 # *IMPORTANT NOTE*: comment temporarily to disambiguate ACEStream <-> ACENetwork
 #                   interdependencies
-add_dependencies (test_u HTTP_GET)
+#add_dependencies (test_u HTTP_GET)
 # *IMPORTANT NOTE* /END
 
 cmake_policy (SET CMP0012 NEW)
