 src/modules/CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/modules/CMakeLists.txt b/src/modules/CMakeLists.txt
index 64c6b1c6..7616f470 100644
--- a/src/modules/CMakeLists.txt
+++ b/src/modules/CMakeLists.txt
@@ -9,7 +9,7 @@ add_subdirectory (misc)
 add_subdirectory (ml)
 # *IMPORTANT NOTE*: comment temporarily to disambiguate ACEStream <-> ACENetwork
 #                   interdependencies
-add_subdirectory (net)
+#add_subdirectory (net)
 # *IMPORTANT NOTE* /END
 add_subdirectory (stat)
 add_subdirectory (vis)
 test_i/CMakeLists.txt | 20 ++++++++++----------
 1 file changed, 10 insertions(+), 10 deletions(-)

diff --git a/test_i/CMakeLists.txt b/test_i/CMakeLists.txt
index 86a13ba4..ce9e20de 100644
--- a/test_i/CMakeLists.txt
+++ b/test_i/CMakeLists.txt
@@ -299,12 +299,12 @@ if (BUILD_TEST_I)
  add_subdirectory (speechcommand)
 # *IMPORTANT NOTE*: comment temporarily to disambiguate ACEStream <-> ACENetwork
 #                   interdependencies
- add_subdirectory (camstream)
- add_subdirectory (filestream)
- add_subdirectory (http_get)
- add_subdirectory (http_get_2)
- add_subdirectory (pop_receive)
- add_subdirectory (smtp_send)
+# add_subdirectory (camstream)
+# add_subdirectory (filestream)
+# add_subdirectory (http_get)
+# add_subdirectory (http_get_2)
+# add_subdirectory (pop_receive)
+# add_subdirectory (smtp_send)
 # *IMPORTANT NOTE* /END
 
  add_dependencies (test_i avsave camera_ar camera_ml camera_msa commandspeech extract_stream imagesave mp3player speechcommand)
@@ -317,10 +317,10 @@ if (BUILD_TEST_I)
 
 # *IMPORTANT NOTE*: comment temporarily to disambiguate ACEStream <-> ACENetwork
 #                   interdependencies
- add_dependencies (test_i camsource camtarget filesource filetarget HTTP_get_1 HTTP_get_2 POP_receive SMTP_send)
- if (WIN32)
-  add_dependencies (test_i camtarget_source camtarget_source_mf)
- endif (WIN32)
+# add_dependencies (test_i camsource camtarget filesource filetarget HTTP_get_1 HTTP_get_2 POP_receive SMTP_send)
+# if (WIN32)
+#  add_dependencies (test_i camtarget_source camtarget_source_mf)
+# endif (WIN32)
 # *IMPORTANT NOTE* /END
 endif (BUILD_TEST_I)
 
 test_u/CMakeLists.txt | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/test_u/CMakeLists.txt b/test_u/CMakeLists.txt
index 857a779a..2249c0fa 100644
--- a/test_u/CMakeLists.txt
+++ b/test_u/CMakeLists.txt
@@ -316,7 +316,7 @@ if (BUILD_TEST_U)
  add_subdirectory (filecopy)
 # *IMPORTANT NOTE*: comment temporarily to disambiguate ACEStream <-> ACENetwork
 #                   interdependencies
- add_subdirectory (http_get)
+# add_subdirectory (http_get)
 # *IMPORTANT NOTE* /END
  add_subdirectory (imagescreen)
  add_subdirectory (mic_visualize)
@@ -329,7 +329,7 @@ if (BUILD_TEST_U)
  add_dependencies (test_u animated_gif audioeffect branch camera_filter camerascreen camsave capturewindow document filecopy imagescreen mic_visualize mp4_player parser QR_decode RIFF_decoder tree)
 # *IMPORTANT NOTE*: comment temporarily to disambiguate ACEStream <-> ACENetwork
 #                   interdependencies
- add_dependencies (test_u HTTP_GET)
+# add_dependencies (test_u HTTP_GET)
 # *IMPORTANT NOTE* /END
 endif (BUILD_TEST_U)
 
